balboa-kafka/src/main/scala/com/socrata/balboa/kafka/Main.scala

```diff
@@ -0,0 +1,133 @@
+package com.socrata.balboa.kafka
+
+import java.io.{IOException, File, FileReader}
+import java.util.Properties
+import java.util.concurrent.Executors
+
+import com.socrata.balboa.kafka.BalboaConfig
+import com.socrata.balboa.kafka.consumer.{DSConsumer, TestConsumer}
+import com.socrata.balboa.metrics.config.Configuration
+import com.socrata.balboa.metrics.data.{DataStoreFactory, DataStore}
+import kafka.consumer.ConsumerConfig
+
+/**
+ * Created by Michael Hotan, michael.hotan@socrata.com on 2/17/15.
+ *
+ * Balboa Kafka Main Application:
+ *
+ * Entry point for Balboa Kafka Consumer.  This application starts a service that listens
+ * on a configurable list of ZooKeeper Host Ports.  This service currently listens for all published Tenant Metrics.
+ *
+ */
+object Main extends App {
+
+  /**
+   * Parse handles command line configuration.
+   */
+  val parser = new scopt.OptionParser[BalboaConfig]("balboa-kafka") {
+    opt[File]('f', "configFile") action { (x,c) => c.copy(configFile = x) } validate {
```

Comment:
Yeah I agree, Currently the way the library I am using is limiting our way of formatting this.  I would love to line break for each rule.